recipe/meta.yaml

```diff
@@ -10,33 +10,46 @@ source:
   sha256: b47aecb16771e1ba0736557255d80ae0240b09156bff434321de559b359ac2d6
 
 build:
-  number: 1
-  skip: true  # [win]
+  number: 2
   script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv '
 
 requirements:
   build:
     - {{ compiler('c') }}
   host:
-    - cython >=0.23
-    - numpy
+    - cython
```

Comment:
`cython` is still contrained: https://github.com/daleroberts/hdmedians/blob/v0.14.2/pyproject.toml#L2

Reply:
Given we are at 0.29, I think we can drop `>=0.23`